gnc/pmc/src/pmc_sim.cc

```diff
@@ -1,14 +1,14 @@
 /* Copyright (c) 2017, United States Government, as represented by the
  * Administrator of the National Aeronautics and Space Administration.
```

Comment:
I think this was merged incorrectly with the nasa/ros2 branch

Reply:
Yes, something went wrong. I think I had pulled your ros2 instead of nasa's. I'll fix this tomorrow and do a PR again